src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {

```

Comment:
Don't begin class or method implementation with an empty line. The class definition should start immediately after the import statements.